generators/php/sdk/src/endpoint/request/WrappedEndpointRequest.ts

```diff
@@ -154,13 +210,14 @@ export class WrappedEndpointRequest extends EndpointRequest {
 
     public getRequestBodyCodeBlock(): RequestBodyCodeBlock | undefined {
         const bodyArgument = this.getRequestBodyArgument();
-        return bodyArgument != null
-            ? {
-                  requestBodyReference: this.serializeJsonRequest({
-                      bodyArgument
-                  })
-              }
-            : undefined;
+        if (bodyArgument == null) {
+            return undefined;
+        }
+
+        return {
+            code: this.getRequestBodyCode(),
+            requestBodyReference: this.serializeJsonRequest({ bodyArgument })
```

Comment:
Is this right if we have a mutlipart request?

Reply:
The serialization code implicitly returned the same thing I sent in but I've added some code to explicitly not call it when we're using multipart